frontend/components/pricing/PremiumCompare.vue

```diff
@@ -47,10 +49,11 @@ const rows = computed(() => {
 
     return {
       value,
-      tooltip
+      tooltip,
+      class: undefined as string | undefined
```

Comment:
that is not pretty, I would suggest to define the return type CompareValue
```
const mapValue = (property: string, perks: PremiumPerks):CompareValue => {
```

then you should be able to remove the whole row here.

Reply:
Obviously better. I couldn't wrap my head around that at first. Thanks!